framework/source/class/qx/ui/basic/Image.js

```diff
@@ -246,6 +246,17 @@ qx.Class.define("qx.ui.basic.Image",
       this.__setSource(el, source);
     },
 
+    // overridden
+    _applyTextColor : function(value)
+    {
+      if (this.__getMode() === "font") {
+        if (value) {
```

Comment:
I guess the content element should be retrieved by
```
var el = this.getContentElement();
if (this.__wrapper) {
  el = el.getChild(0);
}
```
as it's done in e.g. _applyDecorator above.